src/controllers/userController.js

```diff
@@ -0,0 +1,26 @@
+const { catchAsync } = require("../utils/errors");
+const userService = require("../services/userService");
+
+const signin = catchAsync(async (req, res) => {
+  const { KAKAO_CODE } = req.body;
+  const { jwtToken, flag, info } = await userService.signinWithKakao(KAKAO_CODE);
+  return res.status(200).json({ jwtToken, flag, info });
+});
+const signup = async (req, res) => {
+  const user = req.user;
+  const { kakaoId, name, phone, date } = req.body;
+
+  if (!phone || !date || !name) {
+    throw new Error("KEY_ERROR");
+  }
+  const insertId = await userService.signup(kakaoId, name, phone, date);
```

Comment:
```suggestion
  await userService.signup(kakaoId, name, phone, date);
```
insertId라는 변수가 scope내에 어디에도 쓰이지 않는다면 위와 같이 수정해주시는것이 좋습니다!